spec/services/payment_providers/adyen_service_spec.rb

```diff
@@ -58,4 +58,87 @@
       end
     end
   end
+
+  describe '#handle_incoming_webhook' do
+    let(:adyen_provider) { create(:adyen_provider, organization:, hmac_key: nil) }
+
+    let(:body) do
+      JSON.parse(event_response_json)['notificationItems'].first&.dig('NotificationRequestItem')
+    end
+
+    let(:event_response_json) do
+      path = Rails.root.join('spec/fixtures/adyen/webhook_authorisation_response.json')
+      File.read(path)
+    end
+
+    before { adyen_provider }
+
+    it 'checks the webhook' do
+      result = adyen_service.handle_incoming_webhook(
+        organization_id: organization.id,
+        body:
+      )
+
+      expect(result).to be_success
+
+      expect(result.event).to eq(body)
+      expect(PaymentProviders::Adyen::HandleEventJob).to have_been_enqueued
+    end
+
+    context 'when failing to validate the signature' do
+      before do
+        organization.adyen_payment_provider.update! hmac_key: "123"
+      end
+
+      it 'returns an error' do
+        result = adyen_service.handle_incoming_webhook(
+          organization_id: organization.id,
+          body:
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)